soc/renesas/ra/ra2a1/Kconfig

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2024 TOKITA Hiroshi <tokita.hiroshi@fujitsu.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_RA2A1
+	select ARM
+	select CPU_CORTEX_M23
+	select CPU_HAS_ARM_MPU
+	select CPU_CORTEX_M_HAS_VTOR
+	select CPU_CORTEX_M_HAS_SYSTICK
+	select DYNAMIC_INTERRUPTS
+	select TIMER_READS_ITS_FREQUENCY_AT_RUNTIME
+	select XIP
+
+config SOC_OPTION_SETTING_MEMORY
+	bool "Option Setting Memory"
+	default y
+
+config SOC_OPTION_SETTING_LVDAS_DISABLE
+	bool "Disable voltage monitor 0 after reset"
+	default y
+	depends on SOC_OPTION_SETTING_MEMORY
+
+config SOC_OPTION_SETTING_HOCO_DISABLE
+	bool "Disable HOCO oscillator after reset"
+	depends on SOC_OPTION_SETTING_MEMORY
+
+config SOC_OPTION_SETTING_HOCO_FREQ
+	int "HOCO oscillator frequency"
+	default 48000000
+	depends on SOC_OPTION_SETTING_MEMORY
+
+config SOC_OPTION_SETTING_VDSEL
+	int "Voltage Detection 0 Level Select"
+	default 1900
+	depends on SOC_OPTION_SETTING_MEMORY
```

Comment:
Should be dts properties

Reply:
Thank you for reviewing.
This issue discussed in #71017.
I will apply the result of discussions.